packages/create-rspack/template-react-ts/rspack.config.js

```diff
@@ -57,6 +57,6 @@ module.exports = {
 		new rspack.HtmlRspackPlugin({
 			template: "./index.html"
 		}),
-		new refreshPlugin()
-	]
+		isDev ? new refreshPlugin() : null
```

Comment:
NIT: `isDev && new refreshPlugin`

Reply:
Some error. The code in the picture is `isDev && new refreshPlugin()`